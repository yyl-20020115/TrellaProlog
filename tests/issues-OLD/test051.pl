:-initialization(main).

main :-
	a(b,c) =.. [_],
	halt.
main :-
	writeq(false), nl,
	div([-10,0],fois([-4,0],[1,0])) =.. [F,X,Y],
	writeq(F), nl,
	writeq(X), nl,
	writeq(Y), nl,
	writeq(ok), nl.
