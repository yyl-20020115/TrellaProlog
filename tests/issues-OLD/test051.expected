false
div
[-10,0]
fois([-4,0],[1,0])
ok
